Create ui-c8y-1022-37-0-feedback-form-added-to-feature-previews.md (#3779)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-37-0-feedback-form-added-to-feature-previews.md

---
date: ""

> title: Feedback form added to feature previews

Reviewer: ```suggestion
title: Feedback form added to preview features
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64435
version: 1022.37.0
---

> To improve the quality and user experience of new features, a feedback form has been added to feature previews in Cumulocity IoT. When a user opens the feature preview drawer, they will now see a feedback form at the bottom of each feature that allows them to provide their thoughts, suggestions, and any issues encountered. This valuable feedback will help shape and refine the features before they are officially released, ensuring they meet the needs and expectations of our users. Submitting feedback is optional and does not impact the usage of the preview features.

Reviewer: ```suggestion
To improve the quality and user experience of new features, a feedback form has been added for preview features in {{< product-c8y-iot >}}. When opening the **Preview feature** page, a feedback form is now displayed below each feature to provide thoughts, suggestions, and any issues encountered. This feedback will help shape and refine the features before they are officially released, ensuring they meet the needs and expectations of our users. Submitting feedback is optional and does not impact the usage of the preview features.
```

---
date: ""
title: Feedback form added to feature previews.
product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64435
version: 1022.37.0
---
Co-authored-by: Enio Sultan <92023325+eniosultan@users.noreply.github.com>

> Co-authored-by: Carlos Ceia <carlos.ceia@cumulocity.com>

Reviewer 2: ```suggestion
To improve the quality and user experience of new features, a feedback form has been added to feature previews in Cumulocity IoT. When a user opens the feature preview drawer, they will now see a feedback form at the bottom of each feature that allows them to provide their thoughts, suggestions, and any issues encountered. This valuable feedback will help shape and refine the features before they are officially released, ensuring they meet the needs and expectations of our users. Submitting feedback is optional and does not impact the usage of the preview features.
```
